README.md

```diff
@@ -317,6 +317,16 @@ end
 
 ```
 
+In production your compiled JS is cached by each JS VM. In development this cache is cleared whenever you change a `*.jsx*` file in `app/assets/javascripts`. In order to clear this cache when you edit files that do not fit this pattern, you need to tell Rails which paths to watch for changes.
+
+```ruby
+# config/environments/application.rb
+# These are the defaults if you dont specify any yourself
+MyApp::Application.configure do
+  config.watchable_files.concat Dir["#{app.root}/app/assets/javascripts/**/*.jsx*"]
```

Comment:
AFAICT this will only watch files that exist when you boot your development server, new files added after that will not be watched. Wouldn't it make more sense to suggest [watchable_dirs](http://api.rubyonrails.org/classes/Rails/Railtie/Configuration.html#method-i-watchable_dirs) here?